apps/site/assets/css/_page.scss

```diff
@@ -11,6 +11,27 @@
       border-bottom-width: 0;
     }
   }
+
+  .list-group-item {
+    border-top: unset;
+    border-top: solid $gray-lightest;
+    border-width: 1px 0 0 0;
+  }
+
+  .m-event__title {
+    a {
+      color: $black;
```

Comment:
I think `unset` or `inherent` might be a better value here, as it will take on whatever text color is being used in the parent elements (not strictly committing us to `$black`)